firebase.json

```diff
@@ -178,333 +178,8 @@
       { "source": "/research-signup", "destination": "https://docs.google.com/forms/d/e/1FAIpQLSe0i4De809KXVCdljGKrjMj3lxhuzbuFKCtY5PEQPCYtGxFMg/viewform?usp=sf_link", "type": 301 },
       { "source": "/resources/design-principles", "destination": "https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo", "type": 301 },
       { "source": "/style-guide*", "destination": "https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo", "type": 301 },
-      { "source": "/youtube", "destination": "https://youtube.com/@flutterdev", "type": 301 },
+      { "source": "/youtube", "destination": "https://youtube.com/@flutterdev", "type": 301 }
 
-      { "source": "/go/2D-Foundation", "destination": "https://docs.google.com/document/d/1C2hAq-gMAIx4Cbym7EMhVAcPZ2hmNxlgijpYmgTpmUU/edit?usp=sharing", "type": 301 },
```

Comment:
I know this is a draft, but as the PR stands, I have some concerns about removing the existing go links. With them removed, links to design docs around the internet, Discord, etc will be broken. I believe a few don't link to design docs as well. Perhaps we can deprecate the process, but keep the existing links around for a certain mount of time or indefinitely?

Reply:
I am 1000% fine with this, this commit was me exploring what could be causing the breakage in the action.  Will revert!

Reply:
Sounds good, thanks!